packages/api-page-builder/src/plugins/pageContentModifiers.ts

```diff
@@ -14,41 +36,31 @@ export default [
         type: "pb-page-element-modifier",
         elementType: "*",
         getStorageValue({ element }) {
-            const file = get(element, "data.settings.background.image.file.id");
-            if (file) {
-                set(element, "data.settings.background.image.file", file);
-            }
+            const file = element?.data?.settings?.background?.image?.file;
+            element.data.settings.background.image.file = createFileForDatabase(file);
```

Comment:
@brunozoric line 39 uses optional chaining, but line 40 assigns directly to the deeply nested key. Maybe we should use `set` to assign file value?